### PR TITLE
Development

### PR DESCRIPTION
**Pull request by asandan8106**
Hi there!!! This is my To Do List Applications
featuring node.js server, include task management tools (add, delete etc..)

### CODE REVIEW

**Comment by Pali-Pali** [comment]
A more detailed README file would be beneficial. It should include setup instructions, usage guidelines, screenshots, and a list of current features.

**Review by RolivhuwaMuzila — APPROVED** [review]
Well done, I am proud of the changes.

File: public/styles.css
Comment (atlehang66) [inline public/styles.css:1]: Task Complete, another one in the bag!!! Well done @asandan8106

**Review by atlehang66 — APPROVED** [review]
Notes section fixed. I am happy with the current state of the project.

**Pull request merged**